src/main/java/io/jenkins/docker/connector/DockerComputerJNLPConnector.java

```diff
@@ -43,22 +42,14 @@ public class DockerComputerJNLPConnector extends DockerComputerConnector {
     @CheckForNull
     private String user;
 
-    private final JNLPLauncher jnlpLauncher;
```

Comment:
Note that removing this (and the usage in `beforeContainerCreated` etc.) would be incompatible for anyone who was actually using the `tunnel` field using GUI or otherwise XML config. I suspect this option is rarely used (specialized network configuration). Could be restored for compatibility if it seems important.

Other much more obvious fields of `JNLPLauncher`—`webSocket`, `workDir`—were not interpreted at all, so anyone requiring them would I guess have had to put the corresponding options directly into `entryPointArguments` (as they would now need to to with `-tunnel`).